runtime/fiber.c

```diff
@@ -61,6 +61,8 @@ uintnat caml_get_init_stack_wsize (void)
   return stack_wsize;
 }
 
+
+CAMLno_user_tsan /* Disable TSan reports from this function (see #11040) */
```

Comment:
My take-away from #11040 is that those TSan reports hide actual programming errors. In this case, the solution would not be to use `CAMLno_user_tsan`, but to fix them in a separate PR. (Idem for other mentions of #11040 below.)

Reply:
Sorry, I failed to make this clear. Silencing those reports is only a temporary measure until the problems in #11040 are fixed, which we attempt to start doing in #12030.

Reply:
What is the best solution to keep track of the CAMLno_user_tsans that should eventually be removed after fixing the programming errros? Perhaps with an issue with a check list? Or `TODO` comments in the code?